src/App.js

```diff
@@ -1,25 +1,21 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import { BrowserRouter as Router, Switch } from "react-router-dom";
+
+import PublicRoute from "./routes/PublicRoute";
+
+import Login from "./pages/Login/Login";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <div className="wrapper">
+        <div className="main wrap-content">
+          <Switch>
+            <PublicRoute exact path="/login" component={Login} />
```

Comment:
khúc ni cu Hiếu chỉnh thành ri thử nè:

```suggestion
            <PublicRoute exact path="/login" component={() => <Login />} />
```

anh nghĩ làm như ri là mình có thể truyền prop vô cho Login nếu cần á.